cluster.example.yml

```diff
@@ -60,3 +60,9 @@ addons:
     interval: 7d
   kured:
     enabled: true
+cluster:
```

Comment:
I think this is unnecessary padding (the file itself is a cluster level information). Maybe we could just have `name` and `kubeconfig` as top level properties.